### ISSUE TITLE
Existing vis2 project can not be renamed

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.vis#666]
**Describe the bug**  
An existing vis2 project can not be renamed

**To Reproduce**  
1. Go to "Projekte"
2. Click on edit the projct
3. Type a new name of the project

**Expected behavior**  
Project can be renamed.

**Screenshots & Logfiles**  
```
Cannot rename: TypeError: this.socket.rename is not a function [instrument.ts:129:35](http://192.168.178.77:8082/vis-2-beta/widgets/materialdesign/lib/utils/src/instrument.ts)
    ft instrument.ts:129
    setupOnce captureconsole.ts:73
    alert App.jsx:217
    nt App.jsx:1355
    Se 188.70252e45.chunk.js:33
    Se 188.70252e45.chunk.js:33
    nt App.jsx:1345
    rename ProjectDialog.jsx:43
    onClick IODialog.jsx:37
    m1 main.edcbd397.js:1904
    Z5 main.edcbd397.js:1904
    b5 main.edcbd397.js:1904
    a6 main.edcbd397.js:1904
    Z6 main.edcbd397.js:1904
    P3 main.edcbd397.js:1904
    Fa main.edcbd397.js:1907
    g1 main.edcbd397.js:1904
    P3 main.edcbd397.js:1904
    ze main.edcbd397.js:1904
    Q5 main.edcbd397.js:1904
    sentryWrapped helpers.ts:87
```

**Versions:**  
 - Adapter version: vis2 v2.0.18

**Additional context**  
![grafik](https://user-images.githubusercontent.com/25422320/222830326-8e5db5f2-7162-4169-9446-868af29bfa67.png)

**Comment by GermanBluefox** [comment]
Please retest with vis 2.0.23 (Be sure you have latest web adapter 6.0.1 installed)

**Comment by Feuer-sturm** [comment]
issue is fixed with vis2 2.0.33 and web 5.5.3
